### ISSUE TITLE
Webpack bundling issue: "Module not found: Error: Can't resolve 'fs'..."

### ISSUE THREAD

**Issue by lucasconstantino** [Urigo/graphql-modules#360]
Hey guys,

Trying to bundle up my webpack based application with `@graphql-modules/core` included on a _browser_ environment, and I'm get multiple resolver issues like the following:

```
ERROR in ./node_modules/fs.realpath/index.js
Module not found: Error: Can't resolve 'fs' in '/home/luiz/www/lua/spacex/node_modules/fs.realpath'
 @ ./node_modules/fs.realpath/index.js 8:9-22
 @ ./node_modules/glob/glob.js
 @ ./node_modules/graphql-toolkit/dist/sonar/file-scanner.js
 @ ./node_modules/graphql-toolkit/dist/sonar/index.js
 @ ./node_modules/graphql-toolkit/dist/index.js
 @ ./node_modules/@graphql-modules/core/dist/graphql-module.js
 @ ./node_modules/@graphql-modules/core/dist/index.js
 @ ./src/api/schema.js
 @ ./src/api/client.js
 @ ./src/main.js
 @ multi react-hot-loader/patch webpack-dev-server/client?https://channel-dev.tenzen.com.br webpack/hot/only-dev-server ./src/main.js
```

Following the stack tree, it looks like `graphql-toolkit`, recently introduced, is not ready for usage outside Node environments.

I know this is quite an uncommon need, but as the whole Apollo and GraphQL stack are isomorphic (do we still use that term?), maybe `graphql-modules` should be too.

**Comment by ardatan** [comment]
Yes, you're right. Now, we are working on making GraphQL-Modules tree-shakable together with graphql-toolkit. So, it is still WIP.

**Comment by lucasconstantino** [comment]
@ardatan I'm currently trying some downgrade to see if there is any working version. If I have any luck I'll bring it here ;)

**Comment by ardatan** [comment]
We did some changes here; https://github.com/Urigo/graphql-modules/commit/ce5f224eeeabeade6a02475a78e16df00164d250
Could you test with this version;
```
Changes:
 - @graphql-modules/core: 0.4.2 => 0.5.0-alpha.ce5f224e
 - @graphql-modules/di: 0.4.2 => 0.5.0-alpha.ce5f224e
 - graphql-modules: 0.4.2 => 0.5.0-alpha.ce5f224e
```

**Comment by ardatan** [comment]
Fixed in `0.6.0`
